app/src/main/java/com/framgia/englishconversation/screen/conversationdetail/ConversationDetailViewModel.java

```diff
@@ -57,17 +58,41 @@
     private ConversationDetailContract.Presenter mPresenter;
     private FragmentManager mManager;
     private CommentFragment mFragment;
+    private UserModel mTimelineUser;
+    private OnTimelineItemTouchListener mTouchListener =
+            new OnTimelineItemTouchListener<TimelineModel>() {
+                @Override
+                public void onItemTimelineClick(TimelineModel item) {
+                    //TODO
+                }
+
+                /**
+                 * @param item truyền vào khi người dùng click vào layout header item
+                 * check điều kiện nếu userModel từ profile gừi sang trùng với user người tạo
+                 * post thì không điều hướng sang activity profile mới
+                 */
+                @Override
+                public void onItemUserNameClick(TimelineModel item) {
+                    if (mTimelineUser != null && mTimelineUser.getId()
+                            .equals(item.getCreatedUser().getId())) {
+                        return;
+                    }
+                    mDetailActivity.startActivity(ProfileUserActivity.getInstance(mDetailActivity,
+                            item.getCreatedUser()));
+                }
```

Comment:
ERROR: 'method def rcurly' have incorrect indentation level 16, expected level should be one of the following: 8, 12.